concepts/array-destructuring/about.md

```diff
@@ -84,15 +86,14 @@ fourth;
 
 The array destructuring assignment can provide _default values_ in case there is none in the source array.
 
-```
-const [first, second, third="none selected", fourth] = pickAtLeastOne
+```javascript
+const [first, second, third = 'none selected', fourth] = pickAtLeastOne;
 
-third
+third;
 // => 'none selected'
 
-fourth
+fourth;
 // => undefined
 ```
 
-See: `rest-elements`
-See: `spread-elements`
+See: [concept:javascript/rest-and-spread]()
```

Comment:
The widget will render as a box so I think the "See" part needs to be removed otherwise it will probably look strange.